onnxscript/ir/_external_data_test.py

```diff
@@ -115,7 +116,10 @@ class OffloadExternalTensorTest(unittest.TestCase):
 
     def setUp(self):
         # File paths
-        self.temp_dir = tempfile.TemporaryDirectory()  # pylint: disable=consider-using-with
+        if sys.version_info == 3 and sys.version_info >= 10:
```

Comment:
Is this right? `sys.version_info == 3` and `sys.version_info >= 10` cannot be both true?